svg.panzoom.js

```diff
@@ -129,16 +140,18 @@ SVG.extend(SVG.Doc, SVG.Nested, {
       lastP = currentP
     }
 
-    this.on('wheel', zoom)
+    this.on('wheel', wheelZoom)
     this.on('touchstart', pinchZoomStart, this, {passive:false})
     this.on('mousedown', panStart, this, {passive:false})
     this.on('touchstart', panStart, this, {passive:false})
 
+    this.zoom = zoom
```

Comment:
not a fan of this one. This should be added as a real method with extend and not monkey patching it